README.md

```diff
@@ -72,3 +73,45 @@ To show the full stack trace for error occurred during training or testing
 ```bash
 HYDRA_FULL_ERROR=1 python train.py
 ```
+
+## How to Process a PDF
+###  Setup Doc2Json
+To setup Doc2Json, you should run:
+```bash
+sh bin/doc2json/scripts/run.sh
+```
+This will setup Doc2Json and Grobid. And after installation, it starts the Grobid server in the background by default.
+
+
+### How to Parse Reference Strings
```

Comment:
The following part should be in example usage. Make it simple and clear. Details can be leave to README_details.md